Dockerfile

```diff
@@ -105,7 +105,7 @@ RUN luet install -y \
     toolchain/yip \
     toolchain/luet \
     utils/installer \
-    system/cos-setup \
+    system/cos-setup@0.4+6 \
```

Comment:
Isn't `utils/installer` supposed to be updated?

Reply:
oops I pinned the wrong package. fixing